alchemist/alchemist-incarnation-protelis/src/test/resources/advancedorig/converge.pt

```diff
@@ -0,0 +1,9 @@
+module advanced:converge
```

Comment:
Why is this in "advancedorig"? Is this a leftover?

Reply:
It isn't a leftover. I use the file for tests about simulations execution with dynamic classpath. I place files on the correct path dynamically (I create a temp dir). Do i have to change package name?

Reply:
Oh I see.